lib/nosniff.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * X-Content-Type-Options
+ * https://blogs.msdn.microsoft.com/ie/2008/09/02/ie8-security-part-vi-beta-2-update/
+ */
+module.exports = function nosniff(options) {
+    options = options || {};
+
+    // `enabled` should be either `1` or `0`
+    var enabled = (options.enabled !== undefined) ? +options.enabled : 1;
+    
+    return function xssProtection(req, res, next) {
+        if (enabled) {
```

Comment:
Was trying to conform it to the rest of the code base, wasn't quite sure how it all worked out.  Does the enabled part not refer to setting it true/false in config?